server/fishtest/rundb.py

```diff
@@ -617,6 +617,15 @@ def calc_itp(self, run):
         a = 3 # max bonus 1.67x
         itp *= (llr + a) / a
 
+        # Gentle bonus for high game count (up to double the average)
+        if "sprt" in run["args"]:
+            r = run["results"]
+            n = r["wins"] + r["losses"] + r["draws"] + r["crashes"] + r["time_losses"]
```

Comment:
I think this should be just wins + losses + draws (crashes and time losses are part of those).

Reply:
ah yes. dub had that change and I didn't work out why so only got half-way there.
Changed, squashed and force pushed